web-api/storage/scripts/loadTest/loadTestPdfGeneration.ts

```diff
@@ -10,22 +11,55 @@ import axios from 'axios';
 
   const token = await getUserToken({
     cognito,
-    env: process.env.ENV,
-    password: process.env.DEFAULT_ACCOUNT_PASS,
+    env: process.env.ENV || '',
+    password: process.env.DEFAULT_ACCOUNT_PASS || '',
     username: 'petitionsclerk1@example.com',
   });
 
+  const checkPDFComplete = async (asyncSyncId: string) => {
+    await new Promise(resolve => setTimeout(() => resolve(null), 1000));
+    const URL = `https://api-${process.env.DEPLOYING_COLOR}.${process.env.EFCMS_DOMAIN}/results/fetch/${asyncSyncId}`;
+    const headers = {
+      Asyncsyncid: asyncSyncId,
+      Authorization: `Bearer ${token}`,
+      'x-test-user': 'true',
+    };
+
+    const results: {
+      data: {
+        response: any;
+      };
+    } = await axios.get(URL, {
+      headers,
+    });
+
+    if (!results) return await checkPDFComplete(asyncSyncId);
+    const { response } = results.data;
+    if (!response) return await checkPDFComplete(asyncSyncId);
```

Comment:
Is there any issue constantly calling checkPDFComplete rather than assigning a brief wait time between calls?

Reply:
🤦 Thanks!